next/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bratislava-next",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
No need to change this in FE projects. :)

Reply:
Ok, I will take note for any future changes